File: solutions/javascript/raindrops/1/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (num) => {
  if ((num%3 !== 0)&&(num%5 !== 0)&&(num%7 !== 0)){
    return num.toString();
  }
  const div3 = num%3 === 0 ? "Pling":'';
  const div5 = num%5 === 0 ? "Plang":'';
  const div7 = num%7 === 0 ? "Plong":'';
  return div3 + div5 + div7;
};

File: solutions/python/raindrops/1/raindrops.py
def convert(n):
    s=''
    if n%3==0:
        s += "Pling"
    if n%5==0:
        s += "Plang"
    if n%7==0:
        s += "Plong"
    if n%3 and n%5 and n%7: return str(n)
    return s
